### ISSUE TITLE
[RadioGroup] Radio buttons not working

### ISSUE THREAD

**Issue by HadiSDev** [chakra-ui/chakra-ui#975]
**Describe the bug** A clear and concise description of what the bug is.

Radio buttons do not vertically align by default and have very weird margins.

Also when selecting another value, the old value does not reset, leading to multiple selected radio buttons ??

![image](https://user-images.githubusercontent.com/10367624/85627922-94112c80-b66f-11ea-925b-3b42a65b0919.png)

**Link to minimal reproduction** Please provide a codesandbox link or GitHub
repo with a minimal reproduction of the issue.

**Steps to Reproduce** Clear and concise reproduction instructions are important
for us to be able to triage your issue in a timely manner.

https://codesandbox.io/s/reverent-carson-y6bk2?file=/src/App.tsx

**Comment by segunadebayo** [comment]
@HadiSDev, you're using a pre-release version of Chakra UI. Kindly use the `0.8` version and it should work as expected.

For the pre-release, we're updating the docs to reflect some changes we made.

Thanks

**Comment by segunadebayo** [comment]
For the next version, here's how to handle the Radio buttons:

```jsx
<RadioGroup onChange={value => {...}} value={...}>
    <Stack direction="row">
      <Radio value="1">First</Radio>
      <Radio value="2">Second</Radio>
      <Radio value="3">Third</Radio>
    </Stack>
  </RadioGroup>
```

**Comment by HadiSDev** [comment]
@segunadebayo  I will go back to 0.8 but the code you provided still dont work for me. Yes it is now stacking them correctly but when I select 1 radio and then another the first one is still selected: https://codesandbox.io/s/reverent-carson-y6bk2?file=/src/App.tsx

**Comment by euanmillar** [comment]
I was having the same issue.  It only started working after removing the Stack component in between the RadioGroup and Radio component.

This works:  https://codesandbox.io/s/kind-chandrasekhar-xpmlc?file=/src/app.js:290-423

**Comment by akshitarora921** [comment]
@HadiSDev if you want them inline you can as={Flex} to RadioGroup
[code sandbox link](https://codesandbox.io/s/admiring-lovelace-j6b1p?file=/src/index.js)

**Comment by zaferkadi** [comment]
This problem still exists in v1.1.5, and up until v1.1.0 radio group was working fine, not sure why it has not been addressed yet.
https://codesandbox.io/s/chakra-playground-forked-qsf05

**Comment by baohuynhlam** [comment]
Yes I have the same problem as @zaferkadi and it is quite annoying. The radios are very inconsistent. I sometimes found that they actually works, but that same piece of code 10 minutes later after restarting server does not work any more - multiple values can be selected although it is radios.

**Comment by with-heart** [comment]
Should be resolved as of `@chakra-ui/react@1.1.6`: https://github.com/chakra-ui/chakra-ui/pull/3125 

Here's the example from @zaferkadi in `1.1.6`: https://codesandbox.io/s/chakra-playground-forked-udfrd?file=/src/App.js
